src/util/options.rs

```diff
@@ -416,14 +416,24 @@ impl FromStr for NurserySize {
 
 impl Options {
     /// Return upper bound of the nursery size (in number of bytes)
-    pub fn get_max_nursery(&self) -> usize {
+    pub fn get_max_nursery_bytes(&self) -> usize {
         self.nursery.max
     }
 
+    /// Return upper bound of the nursery size (in number of pages)
+    pub fn get_max_nursery_pages(&self) -> usize {
+        crate::util::conversions::bytes_to_pages_up(self.nursery.max)
+    }
+
     /// Return lower bound of the nursery size (in number of bytes)
-    pub fn get_min_nursery(&self) -> usize {
+    pub fn get_min_nursery_bytes(&self) -> usize {
         self.nursery.min
     }
+
+    /// Return lower bound of the nursery size (in number of bytes)
```

Comment:
Should be number of pages:

```suggestion
    /// Return lower bound of the nursery size (in number of pages)
```